flutter/ios/Classes/SwiftSentryFlutterPlugin.swift

```diff
@@ -3,27 +3,257 @@ import Sentry
+
+                switch (diagnosticLevel) {
+                case "fatal", "error":
+                    options.logLevel = .error
+                    break;
+                case "debug":
+                    options.logLevel = .debug
+                    break;
+                case "warning", "info":
+                    options.logLevel = .verbose
+                    break;
+                default:
+                    options.logLevel = .none
+                    break;
+                }
+            }
+
+            if let sessionTrackingIntervalMillis = arguments["sessionTrackingIntervalMillis"] as? UInt{
+                options.sessionTrackingIntervalMillis = sessionTrackingIntervalMillis
+            }
+
+            if let dist = arguments["dist"] as? String{
+                options.dist = dist
+            }
+
+
+            if let enableAutoNativeBreadcrumbs = arguments["enableAutoNativeBreadcrumbs"] as? Bool,
+               enableAutoNativeBreadcrumbs == false {
+                options.integrations = options.integrations?.filter { (name) -> Bool in
+                    return name != "SentryAutoBreadcrumbTrackingIntegration"
+                }
+            }
+
+            if let maxBreadcrumbs = arguments["maxBreadcrumbs"] as? UInt{
+                options.maxBreadcrumbs = maxBreadcrumbs
+            }
+
+            print("------ options.integrations =======> \(String(describing: options.integrations))")
+            //print("------ packages =======> \(String(describing: arguments["packages"] as? [String]))")
+            //print("------ integrations =======> \(String(describing: arguments["integrations"] as? [String]))")
+
+            self.sentryOptions = options
+            /*
+             TODO : beforeSend alternative =>
+              - eventOrigin
+             */
+
+            /* for now, in sentry-cocoa, beforeSend is not called before captureEnvelope
+            options.beforeSend = { event in
+                self.setEventOriginTag(event: event)
+
+                if let sdk = event.sdk, self.isValidSdk(sdk: sdk){
+                    if let packages = arguments["packages"] as? [String]{
+                        if  var sdkPackages = sdk["packages"] as? [String]{
+                            event.sdk!["packages"] = sdkPackages.append(contentsOf: packages)
+                        } else {
+                            event.sdk = ["packages":packages]
+                        }
+                    }
+
+                    if let integrations = arguments["integrations"] as? [String]{
+                        if  var sdkIntegrations = sdk["integrations"] as? [String]{
+                            event.sdk!["integrations"] = sdkIntegrations.append(contentsOf: integrations)
+                        } else {
+                            event.sdk = ["integrations":integrations]
+                        }
+                    }
+                }
+                return event
+            }*/
```

Comment:
same as https://github.com/getsentry/sentry-dart/pull/161/files#r526670966

if events are raised on Dart/Flutter, we need to append integrations and packages before capturing the envelope.
if events are raised on iOS, we can append them here, on beforeSend

Reply:
✅ done, needs tests